editor/animation_track_editor.cpp

```diff
@@ -7121,7 +7121,7 @@ void AnimationTrackEditor::_pick_track_select_recursive(TreeItem *p_item, const
 	NodePath np = p_item->get_metadata(0);
 	Node *node = get_node_or_null(np);
 
-	if (node && !p_filter.is_empty() && ((String)node->get_name()).findn(p_filter) != -1) {
+	if (node && !p_filter.is_empty() && ((String)node->get_name()).containsn(p_filter)) {
```

Comment:
IIUC we also have this in StringName, so this should work?
```suggestion
	if (node && !p_filter.is_empty() && node->get_name().containsn(p_filter)) {
```

Reply:
Will test

Reply:
It's only available to `StringName` in GDScript, so the cast is necessary